tap_twitter_ads/schemas/campaigns.json

```diff
@@ -8,10 +8,6 @@
     "name": {
       "type": ["null", "string"]
     },
-    "start_time": {
```

Comment:
Any reason of removing these fields?

Reply:
`start_time` and `end_time` fields are deprecated. ([link](https://twittercommunity.com/t/ads-api-version-11/168814#changed-moving-start-end-times-to-line-item-level-6)) That's why removed it from the schema.

Reply:
Got it, make sure you added those fields inside Line Item.